exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -408,5 +448,47 @@ public void close() throws Exception {
     if (embeddedJetty != null) {
       embeddedJetty.stop();
     }
+    //Deleting temp directory
+    FileUtils.deleteDirectory(getTmpJavaScriptDir());
```

Comment:
Will this fail if we unable to delete temp dir?

Reply:
My guess is that deleting a non-empty directory without this would not work. This should recursively go and clean up.

Reply:
I meant what if we fail to delete temp directory? May be use delete quietly.

Reply:
Again defending `getTmpJavaScriptDir()` usage is incorrect. Consider the case when server start will fail before generating js temp dir, so in close method you'll call `getTmpJavaScriptDir()` and it will generate js options since it did not do it at start up since all we need is to delete the directory.

Reply:
I wanted to generate the JS lazily, hence this approach. I'll make the necessary change and generate it on startup.